scripts/backend_test_shards.json

```diff
@@ -0,0 +1,286 @@
+{
+    "1": [
```

Comment:
How expensive and bad (compare to what we have in this PR) it would be if we will autogenerate the shard using the length of the test files? (Just wanted to check whether we have a way to avoid maintaining a new JSON file.)

Reply:
I think we want manual control over the shards. Test file length might not actually be a very good proxy for test runtime